README.md

```diff
@@ -1070,6 +1078,7 @@ It is also a good practice to annotate overridden methods with `@Override` to ma
 
 * Firebase. [Firebase.google.com](https://firebase.google.com/)
 
+* Leak Canary [Droidcon NYC 2015 About Leak Canary by Py](https://www.youtube.com/watch?v=mU1VcKx8Wzw)
```

Comment:
I would add the actual link 
https://square.github.io/leakcanary/

Reply:
okay sure

Reply:
add it